acctest/paracctest/test-fixtures/ds.pkr.hcl

```diff
@@ -0,0 +1,29 @@
+
+variable "bkt" {
+    default = "pkr-acctest-temp-1"
+}
+
+data "hcp-packer-iteration" "acc" {
+    bucket_name = var.bkt
+    channel     = "acc"
+}
+
+data "hcp-packer-image" "acc-production" {
+    bucket_name    = var.bkt
+    iteration_id   = data.hcp-packer-iteration.acc.id
+    cloud_provider = "aws"
+    region         = "us-west-1"
+}
+
+source "null" "example" {
```

Comment:
If you wanted a builder that created an image of some sort you can also use the file builder for now.

Reply:
doh, I didn't check that one !